pkg/tss/upload/out/etc/group

```diff
@@ -0,0 +1,6 @@
+root:x:0:root
+bin:x:1:root,bin,daemon
+daemon:x:2:root,bin,daemon
+sys:x:3:root,bin,adm
+adm:x:4:root,adm,daemon
+tss:x:116:
```

Comment:
I don't like having `upload/out` in the linuxkit tree. Usually we just put things directly under `etc/` say in the build tree and the `Dockerfile` puts things in the right place; if that is really inconvenient, then something like `upload` would be ok, but not with `out` under it as that is only there because that is an internal directory you are using in the build process.